composer.json

```diff
@@ -1,8 +1,8 @@
 {
-    "name": "websitetoolbox/websitetoolboxforum",
-    "description": "Your CraftCMS forum doesn’t need to be basic. Website Toolbox is a cloud-based forum plugin that allows you to easily add a powerful, intuitive, and maintenance-free forum to CraftCMS without slowing it down. No database, servers, or coding required",
+    "name": "websitetoolbox/websitetoolboxcommunity",
+    "description": "Your CraftCMS forum doesn’t need to be basic. Website Toolbox is a cloud-based community plugin that allows you to easily add a powerful, intuitive, and maintenance-free community to CraftCMS without slowing it down. No database, servers, or coding required.",
```

Comment:
Again, some places you say forum and some you say community. Changes in this pull request are so sloppy and inconsistent. Disappointing. These are obvious issues.

@masihur - ???

Reply:
Replaced all forum word with community.

Reply:
"CraftCMS forum" - Change to "CraftCMS community".

Reply:
Change the whole description to this:
The Easiest CraftCMS Community Plugin. Effortlessly Build a Beautiful Community Forum with Zero Coding, Seamless Embedding, Instant Setup, and Exceptional Support.